packages/ezflow_extension/ezextension/ezflow/classes/ezflowfetchinterface.php

```diff
@@ -27,6 +27,13 @@
 interface eZFlowFetchInterface
 {
     /**
+     * Constructor
+     *
+     * @param array $parameters
+     */
+
+    public function __construct( $parameters );
```

Comment:
This would be 1. a bc break, and 2. putting constructors on interfaces is a bit of a no no